octoprint_bettergrblsupport/__init__.py

```diff
@@ -879,59 +890,30 @@ def on_api_command(self, command, data):
             self._logger.debug("move {} {}".format(direction, distance))
 
             if direction == "home":
-                self._printer.commands("G91")
-                self._printer.commands("G28 X0 Y0")
-                self._printer.commands("G90")
+                self._printer.commands("G90 G0 X0 Y0")
 
             if direction == "forward":
-                self._printer.commands("G91")
-                self._printer.commands("G0 Y{}".format(distance))
-                self._printer.commands("G90")
+                self._printer.commands("G91 G0 Y{}".format(distance))
 
             if direction == "backward":
-                self._printer.commands("G91")
-                self._printer.commands("G0 Y{}".format(distance * -1))
-                self._printer.commands("G90")
+                self._printer.commands("G91 G0 Y{}".format(distance * -1))
 
             if direction == "left":
-                self._printer.commands("G91")
-                self._printer.commands("G0 X{}".format(distance * -1))
-                self._printer.commands("G90")
+                self._printer.commands("G91 G0 X{}".format(distance * -1))
 
             if direction == "right":
-                self._printer.commands("G91")
-                self._printer.commands("G0 X{}".format(distance))
-                self._printer.commands("G90")
+                self._printer.commands("G91 G0 X{}".format(distance))
 
             if direction == "up":
-                self._printer.commands("G91")
-                self._printer.commands("G0 Z{}".format(distance))
-                self._printer.commands("G90")
+                self._printer.commands("G91 G0 Z{}".format(distance))
 
             if direction == "down":
-                self._printer.commands("G91")
-                self._printer.commands("G0 Z{}".format(distance * -1))
-                self._printer.commands("G90")
-
-
+                self._printer.commands("G91 G0 Z{}".format(distance * -1))
             return
 
         if command == "origin":
             # do origin stuff
-            self.ignoreErrors = True
```

Comment:
not sure why this was removed.... TBT, I'm not even sure why I put it in there to begin with

Reply:
I think I made this verbose like this because of variations across engravers.   I really do like the streamlined model but need to validate we don't break other engravers